pkg/virt-launcher/virtwrap/device/sriov/hostdev.go

```diff
@@ -193,3 +193,45 @@ func deviceLookup(hostDevices []api.HostDevice, deviceAlias string) *api.HostDev
 	}
 	return nil
 }
+
+type deviceAttacher interface {
+	AttachDeviceFlags(xmlData string, flags libvirt.DomainDeviceModifyFlags) error
+}
+
+func AttachHostDevices(dom deviceAttacher, hostDevices []api.HostDevice) error {
+	var errors []error
```

Comment:
nit: I would use `errs` as `errors` is a known std package.

Reply:
That make sense, I changed it